packages/ui/src/theme/css/component/radio.scss

```diff
@@ -0,0 +1,80 @@
+.amplify-radio {
+  gap: inherit;
+}
+
+.amplify-radio__button {
+  width: var(--amplify-components-radio-button-width);
+  height: var(--amplify-components-radio-button-height);
+  box-sizing: var(--amplify-components-radio-button-box-sizing);
+  border-width: var(--amplify-components-radio-button-border-width);
+  border-style: var(--amplify-components-radio-button-border-style);
+  border-radius: var(--amplify-components-radio-button-border-radius);
+  border-color: var(--amplify-components-radio-button-border-color);
+  color: var(--amplify-components-radio-button-color);
+  background-color: var(--amplify-components-radio-button-background-color);
+  transition-property: var(
+    --amplify-components-radio-button-transition-property
+  );
+  transition-duration: var(
+    --amplify-components-radio-button-transition-duration
+  );
+
+  // For case where box shadow doesn't work
+  // https://stackoverflow.com/questions/65663864/does-replacing-a-form-inputs-outline-with-box-shadow-violate-any-wcag-guideline
+  outline-color: var(--amplify-components-radio-button-outline-color);
+  outline-style: var(--amplify-components-radio-button-outline-style);
+  outline-width: var(--amplify-components-radio-button-outline-width);
+  outline-offset: var(--amplify-components-radio-button-outline-offset);
+
+  &::before {
+    content: var(--amplify-components-radio-button-before-content);
+    display: var(--amplify-components-radio-button-before-display);
+    width: var(--amplify-components-radio-button-before-width);
+    height: var(--amplify-components-radio-button-before-height);
+    border-radius: var(--amplify-components-radio-button-before-border-radius);
+    background-color: var(
+      --amplify-components-radio-button-before-background-color
+    );
+  }
+
+  &[data-size='small'] {
+    width: var(--amplify-components-radio-button-small-width);
+    height: var(--amplify-components-radio-button-small-height);
+  }
+  &[data-size='large'] {
+    width: var(--amplify-components-radio-button-large-width);
+    height: var(--amplify-components-radio-button-large-height);
+  }
+}
+
+.amplify-radio__input:checked + .amplify-radio__button {
+  border-color: var(--amplify-components-radio-button-checked-border-color);
+  background-color: var(
+    --amplify-components-radio-button-checked-background-color
+  );
+}
+
+.amplify-radio__input:focus + .amplify-radio__button {
+  border-color: var(--amplify-components-radio-button-focus-border-color);
+  box-shadow: var(--amplify-components-radio-button-focus-box-shadow);
```

Comment:
We need to bump up the contrast on this focus color. I think the contrast has to be 4.5:1 for the focus color against both page background color and the button background color.

Contrast with button background-color:
<img width="838" alt="Screen Shot 2021-09-13 at 3 04 23 PM" src="https://user-images.githubusercontent.com/6165315/133162600-64ebc1c1-cc89-4c6c-aff9-ef43d16edeab.png">

Contrast with white background:
<img width="394" alt="Screen Shot 2021-09-13 at 3 06 38 PM" src="https://user-images.githubusercontent.com/6165315/133162977-3167c980-b42a-488b-96e1-71b8f9644163.png">

Reply:
One possible way to meet the contrast guideliness would be to make the dot in the middle the dark color like this example:
https://a11y-style-guide.com/style-guide/section-forms.html
![Screen Shot 2021-09-13 at 3 21 11 PM](https://user-images.githubusercontent.com/6165315/133164168-29690d3d-750c-4af2-afdf-bea91d1b8973.png)

Reply:
That's a good callout. Will update